src/SDKs/_metadata/servicebus_resource-manager.txt

```diff
@@ -1,9 +1,9 @@
-2018-03-01 19:04:56 UTC
+2018-03-27 22:33:18 UTC
```

Comment:
Please create a `generate.ps1` script similar to [this](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/generate.ps1) and regenerate the code

Reply:
resolved